src/entry.js

```diff
@@ -15,6 +15,8 @@ if (typeof Components !== 'undefined') {
   // See: https://bugzilla.mozilla.org/show_bug.cgi?id=1007318
   Blob = hiddenWindow.Blob;
   WebSocket = hiddenWindow.WebSocket;
+  FileReader = hiddenWindow.FileReader;
```

Comment:
I can add that. Should I also be adding a test in the freedom repo which will fail without this?